ccd-definition/AuthorisationComplexType/caseDocuments/AdditionalApplicationsBundle.json

```diff
@@ -1493,6 +1493,44 @@
       }
     ]
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "additionalApplicationsBundle",
+    "ListElementCode": "c2DocumentBundle.respondents",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[LABARRISTER]",
```

Comment:
they are not live yet, but you have extras child solicitors that probably will need the access?
          "[CHILDSOLICITORA]",
          "[CHILDSOLICITORB]",
          "[CHILDSOLICITORC]",
          "[CHILDSOLICITORD]",
          "[CHILDSOLICITORE]",
          "[CHILDSOLICITORF]",
          "[CHILDSOLICITORG]",
          "[CHILDSOLICITORH]",
          "[CHILDSOLICITORI]",
          "[CHILDSOLICITORJ]",
          "[CHILDSOLICITORK]",
          "[CHILDSOLICITORL]",
          "[CHILDSOLICITORM]",
          "[CHILDSOLICITORN]",
          "[CHILDSOLICITORO]",

Reply:
ok, I'll add the child solicitor roles